site/templates/items/tagline.jet

```diff
@@ -1,63 +1,48 @@
-{{block metaItemTagline(genres=true, genresLimit=-1, classification=true)}}
-  <ul class="meta-item-tagline">
-
-    {*<li class="meta-item-tagline-item">
-      <span class="meta-item-classification meta-item-classification-mature">M</span>
-    </li>*}
-
-    {{if classification==true}}
-    <li class="meta-item-tagline-item meta-item-tagline-item-classification">
+{{ block metaItemTagline(genres=true, genresLimit=-1, classification=true) }}
+  <div class="meta-item-tagline">
+    {{ if classification==true }}
       <s72-classification-label data-slug="{{.Slug}}" data-layout="tooltip"></s72-classification-label>
-    </li>
-    {{end}}
+    {{ end }}
 
-    {{if genres == true}}
+    {{ if genres == true }}
       {{ if isset(.Genres) && len(.Genres) > 0 }}
-
         {{ itemGenres := .Genres }}
         {{ if genresLimit > -1  && len(itemGenres) > genresLimit }}
           {{ itemGenres = itemGenres[:genresLimit] }}
         {{ end }}
 
-        {{if isset(itemGenres)}}
-          <li class="meta-item-tagline-item">{{itemGenres}}</li>
-        {{end}}
-
-        {{if isset(.Runtime) || isset(.Episodes)}}
-          <li class="meta-item-tagline-divider">•</li>
-        {{end}}
-
-      {{end}}
-
-      {{if isset(.ShowInfo) && isset(.ShowInfo.Genres)}}
-        <li class="meta-item-tagline-item">{{.ShowInfo.Genres}}</li>
-
-        {{if isset(.Runtime) || isset(.Episodes)}}
-          <li class="meta-item-tagline-divider">•</li>
-        {{end}}
-
-      {{end}}
-
-    {{end}}
-
-    {{if isset(.Runtime)}}
-
-      {{if .Runtime > 60}}
-        <li class="meta-item-tagline-item">{{.Runtime.Hours()}}{{i18n("runtime_hours")}} {{.Runtime.Minutes()}}{{i18n("runtime_minutes")}}</li>
-      {{else}}
-        <li class="meta-item-tagline-item">{{.Runtime}}min</li>
-      {{end}}
+        {{ if isset(itemGenres) }}
+          <span class="meta-item-tagline-item">{{ itemGenres }}</span>
+        {{ end }}
 
-      {*<li class="meta-item-tagline-item">{{.Runtime}}min</li>*}
-    {{end}}
+        {{ if isset(.Runtime) || isset(.Episodes) }}
+          <span class="meta-item-tagline-divider">•</span>
+        {{ end }}
+      {{ end }}
 
-    {{if isset(.Items)}}
-      <li class="meta-item-tagline-item">{{i18n("bundle_items_all_films", len(.Items))}}</li>
-    {{end}}
+      {{ if isset(.ShowInfo) && isset(.ShowInfo.Genres) }}
+        <span class="meta-item-tagline-item">{{ .ShowInfo.Genres }}</span>
 
-    {{if isset(.Episodes)}}
-      <li class="meta-item-tagline-item">{{i18n("episode_count", len(.Episodes))}}</li>
+        {{ if isset(.Runtime) || isset(.Episodes) }}
+          <span class="meta-item-tagline-divider">•</span>
+        {{ end }}
+      {{ end }}
+    {{ end }}
+
+    {{ if isset(.Runtime) }}
+      {{ if .Runtime > 60 }}
+        <span class="meta-item-tagline-item">{{ .Runtime.Hours() }}{{ i18n("runtime_hours") }} {{ .Runtime.Minutes() }}{{ i18n("runtime_minutes") }}</span>
```

Comment:
Is there a reason why the jet bits have been fluffed out with delimiter spaces? I assume an editor auto-format? It makes the PR diff hard to notice the actual changes.

I acknowledge [the official docs](https://github.com/CloudyKit/jet/blob/master/docs/syntax.md) 95% support the extra-fluffiness, but this line in particular I find it harder for my eye to spot where there should/shouldn't be a space in the rendered output and it also word wraps in a worse place. Perhaps we should take this to the team for a wider dev discussion and add an editorconfig to enforce a style.

Reply:
I do it because I find it harder to read without the spaces.